spec/traits.dd

```diff
@@ -842,6 +844,24 @@ export
 public
 )
 
+$(H2 $(GNAME getTargetInfo))
+
+        $(P Receives a string key as argument.
+        The result is an expression describing the requested target information.
+        )
+
+---
+version (CppRuntime_Microsoft)
+    static assert(__traits(getTargetInfo, "cppRuntimeLibrary") == "libcmt");
+---
+
+        $(P Keys are implementation defined, allowing relevant data for exotic targets.
+        A reliable subset exists which are always available:
+        )
+
+        $(UL
+        $(LI $(D "cppRuntimeLibrary"))
```

Comment:
Can we document what it means as well?

Reply:
I thought about it briefly, but I wouldn't know how to create a table with 2 columns in ddoc.
It's also astoundingly self-explanatory :)

Reply:
> I thought about it briefly, but I wouldn't know how to create a table with 2 columns in ddoc.

For simplicity I would go with a list (as it currently is) with the name in bold, a dash (or a colon) and then the description, like this:

* **cppRuntimeLibrary** - description

Or if you want to go with a table, you can have a look at how it's done in the algorithm modules in Phobos, they all have a table at the top.

> It's also astoundingly self-explanatory :)

In this case perhaps. BTW, is it the C++ runtime library or C++ standard library. I know Clang has a separate C++ ABI and standard library.